features/saved_searches/update.feature

```diff
@@ -33,8 +33,33 @@ Feature: Test the UDB3 saved searches API
 
     When I send a GET request to "/saved-searches/v3"
     Then the response status should be "200"
-    And the response body should be valid JSON
+#    And the response body should be valid JSON
     And the JSON response should include:
       """
       {"name":"%{name}","query":"Lessen in de avond","id":"%{id}"}
-      """
\ No newline at end of file
+      """
+
+  #Bugfix https://jira.publiq.be/browse/III-6019
+  Scenario: I update a saved search twice
```

Comment:
This makes me wonder if we need to make a test where one user would be updating the saved search of another user?

Reply:
Added a test for this, and also for when the saved search does not exist